tests/unit/translator/resources/variable_init.pml

```diff
@@ -36,8 +36,8 @@ inline Actuator_0_transition(id)
 inline Actuator_0_init()
 {
   MyInteger_assign_value(global_state.actuator.param, 3);
-  MyInteger_assign_value(global_state.actuator.seq.x, 1);
-  MyInteger_assign_value(global_state.actuator.seq.y, 2);
+  MySeq__x_assign_value(global_state.actuator.seq.x, 1);
```

Comment:
Does this imply that there will be many more inlines generated?

Reply:
No, the number of generated inlines is not changed. The generator uses type from left operand to create name of the inline.